content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -34,17 +34,16 @@ for debugging if your cluster offers this.
 {{< note >}}
 While Kubernetes supports more
 {{< glossary_tooltip text="container runtimes" term_id="container-runtime" >}}
-than just Docker, [Docker](https://www.docker.com/) is the most commonly known
-runtime, and it helps to describe Pods using some terminology from Docker.
+than just Docker, Docker is the most commonly known
+runtime.
```

Comment:
This update isn't really removing mention of Docker.

Reply:
hey can it be rephrased as 
note :  Kubernetes supports **( many or more than one )** [container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes) **( like or such as containerd or runC )**

Reply:
I'd expect any fix to include a **substantial** rewrite so that that note is no longer required. That would be more than a one-sentence change.